engine/lib/luajit-ffi-gen/src/impl_item/type_info.rs

```diff
@@ -29,6 +29,7 @@ const COPY_TYPES: &[&str] = &[
     "DVec4",
     "Vec4",
     "Box3",
+    "Position",
```

Comment:
This list is for types that are copyable in LuaJit terms and Position type (as well as other Vec3/4 types) cannot be propery copied by Lua.
We should remove all Vec3/4 and Box3 types from this list.
I have this as a separate task but if you already working on it, can you probably do this cleanup? Unless it causes a lot of refactoring.
At least Position should not be here.

Reply:
I'm going to leave it for now, and deal with this separately. When removing Position here, it seems to cause other problems in luajit-ffi-gen which I can't recall at this moment.